### DISCUSSION TITLE
After adding the note I can't find it anymore

### DISCUSSION THREAD

**Discussion by christian179717** [Helium314/SCEE#551] [Q&A] [answered]
Going around I added some notes of things I want to added at home with my pc. But then I didn't find the notes anymore.
The notes that I've created direct to objects are visible, but the notes I created with long press add note aren't. Can anybody help me?
[Video](https://private-user-images.githubusercontent.com/156656/330595027-cb54acd2-88f7-4e13-a41b-61c3702baa44.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgwMjA4MzEsIm5iZiI6MTcxODAyMDUzMSwicGF0aCI6Ii8xNTY2NTYvMzMwNTk1MDI3LWNiNTRhY2QyLTg4ZjctNGUxMy1hNDFiLTYxYzM3MDJiYWE0NC5tcDQ_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMFQxMTU1MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNGZkYTM2NDA4NmVhODRmZTAzY2NkNWI2MTkzYjEyYWNjNTRlMjRkYTdjYWI3ZGRhMDMyNGY2NmI5Mjc5ZmEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.x2CGRW_KV_yd7TZnxIGHvXX_iKtn915hvs0amWYh-PQ)
This video is from another user but it looks exactly the same, I create the note and then it disappears.

**Comment by mnalis** [comment]
That video does not show to me, unfortunately.

Is it perhaps related that SC(EE) hides notes created by yourself?
- i.e. see https://github.com/streetcomplete/StreetComplete/issues/2692

That has bothered me a lot, but never managed to suggest PR in SCEE to add extra option to fix it...

**Comment by christian179717** [comment]
So you don't have any solution?
Try video please again.

**Reply by riQQ** [reply]
Can't open the video either. Neither the original one nor the one from the edit.

**Reply by christian179717** [reply]
https://github.com/Helium314/SCEE/issues/543#issuecomment-2111167120
Video in this comment

**Comment by Helium314** [comment]
What do you mean with the notes aren't visible?
_Where_ aren't they visible? Are they in your own notes in the OSM profile?

**Reply by christian179717** [reply]
In SCEE with my profile logged in.
I installed SCEE on another device without signing in, and they aren't there visible too.

First I thought they are created in the OSM Database, but in editors like ID, JOSM, Vespucci I don't find them either.

Where can I check my notes in the OSM profile?

**Reply by Helium314** [reply, marked as answer]
Go to osm.org -> my profile -> my notes. All notes you uploaded or commented on should be shown here.

**Reply by christian179717** [reply]
Thank you very much

**Reply by mnalis** [reply]
> In SCEE with my profile logged in.

If you need to see your Notes in SCEE, one workaround is to select in SCEE preferences `Show all notes` and then in `My Profile` choose `Logout`. That should makes even your own notes visible in SCEE (as it doesn't know who you are anymore after logout :smiley: so can't filter them out)

But that is obviously ugly workaround, so I've now (finally) found time to propose https://github.com/Helium314/SCEE/issues/552
